coffee_core/src/coffee.rs

```diff
@@ -285,6 +287,13 @@ impl PluginManager for CoffeeManager {
                         plugin.root_path = new_root_path;
                         plugin.exec_path = new_exec_path;
 
+                        if let Some(branch) = branch {
+                            // FIXME: Where we store the date? how we manage it?
+                            let (commit, _) =
+                                git_checkout(&plugin.root_path, &branch, verbose).await?;
```

Comment:
We switch the whole repository to the required branch but we don't switch back to the original branch of the repository after installing the plugin

Reply:
nice point